widgetastic_patternfly.py

```diff
@@ -618,7 +618,7 @@ def select_by_visible_text(self, *items):
                 './div/ul/li/a[./span[contains(@class, "text") and normalize-space(.)={}]]'.format(
                     quote(text)),
                 parent=self)
-        self.close()
+        # self.close()
```

Comment:
![screenshot from 2017-03-13 09 12 57](https://cloud.githubusercontent.com/assets/3866976/23863552/59f5cc44-07cd-11e7-9e40-fe6d22861f07.png)
Then on selecting data in first screen immediately this screen loads . Here it gives error about not finding Bootstrapselect not found